Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx

```diff
@@ -44,7 +44,7 @@ class SearchPageInput extends Component {
                             value={this.state.searchTerm}
                             onChange={this.onSearchFieldChange.bind(this)}
                             onKeyPress={(e) => { e.key === "Enter" ? this.props.onSearch(this.state.searchTerm) : null; }}
-                            placeholder="Search" />
+                            placeholder="{Localization.get("Search")}" />
```

Comment:
I don't think the double-quotes should be kept here, did you test this out ?

Reply:
No, unfortunately, I could not test. I do not have the test conditions.

Reply:
> I don't think the double-quotes should be kept here, did you test this out ?

I agree 100% regarding the outer quotes.